### PR TITLE
[strings] fi update

### PR DESCRIPTION
**Pull request by scop**

### CODE REVIEW

File: data/strings/strings.txt [hunk 1]
```diff
@@ -12692,7 +12705,8 @@
     ja = 番地
     ko = 집 번호
     lt = Pastato numeris
-    mr = बिल्डिंग क्रमांक
+    mr = बिल्डिंग क्रमांकمشاهده جستجو‌های اخیر
```
Comment (biodranik) [inline data/strings/strings.txt:12708]: Please undo this change.

**Review by biodranik — CHANGES_REQUESTED** [review]
Thank you! One line should be reverted.

File: data/strings/strings.txt (hunk 1 above)
Comment (scop) [reply data/strings/strings.txt:12708]: Oops! Done.

**Review by biodranik — APPROVED** [review]
Kiitos paljon!
